Samples/Com.OneSignal.Sample.iOS/Com.OneSignal.Sample.iOS.csproj

```diff
@@ -32,7 +32,8 @@
     <DebugType>pdbonly</DebugType>
```

Comment:
Changes in this file don't look needed. Can we revert this file?

Reply:
Yes, these can be reverted (VS did them automatically). Is there a simple way to revert specific files to a commit with git in VS?